content/chainguard/chainguard-images/videos/repro.md

```diff
@@ -0,0 +1,236 @@
+---
+title: "Reproducibility and Chainguard Images"
+linktitle: "Reproducibility and Chainguard Images"
+lead: ""
+description: "This video explains the importance of reproducibility and how to reproduce any
```

Comment:
Maybe we could use another word second time around so we don't say "Reproducibility/reproducible" twice? "Recreate" or "reconstruct"?

Reply:
Changed to recreate.